controllers/util/events.go

```diff
@@ -71,24 +75,60 @@ const (
 	// deploys ramen and the application in the managed cluster initially
 	EventReasonDeploySuccess = "DRPCDeploySuccess"
 
+	//  EventReasonCreateVRGManifestWork is generated to create VRG ManifestWork
+	EventReasonCreateVRGManifestWork = "CreateVRGManifestWork"
+
+	//  EventReasonVRGManifestWorkCreationFailed is generated when DRPC failed to
+	// create VRG ManifestWork
+	EventReasonVRGManifestWorkCreationFailed = "CreateVRGManifestWorkFailed"
+
+	// EventReasonUpdateUserPlacementRule is generated to update the user PlacementRule
+	EventReasonUpdateUserPlacementRule = "UpdateUserPlacementRule"
+
+	// EventReasonUpdateUserPlacementRuleFailed is generated when
+	// DRPC fails to update the user PlacementRule
+	EventReasonUpdateUserPlacementRuleFailed = "UpdateUserPlacementRuleFailed"
+
+	// EventReasonDeployFail is generic, can we re-use it with appropriate failure reason?
```

Comment:
EventReasonDeployFail is generic, can we re-use it with appropriate failure message  or use more specific reason like "EventReasonVRGManifestWorkCreationFailed" instead?